### PR TITLE
Mes-8257-cat d-safety-score

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Added safety score to search completed and debrief page, fixed small dropdown visual error on vehicle checks

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image
image

### CODE REVIEW

File: src/app/pages/view-test-result/components/debrief-card/debrief-card.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export class DebriefCardComponent implements OnInit {
   }
 
   ngOnInit() {
+    console.log('Data: ', this.data);
```
Comment (gittins7) [inline src/app/pages/view-test-result/components/debrief-card/debrief-card.ts]: Please remove this log

Comment (RLCorp) [reply src/app/pages/view-test-result/components/debrief-card/debrief-card.ts]: addressed in newest commit

**Review by gittins7 — APPROVED** [review]

File: src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  data: SafetyQuestionResult[];
+
+  @Input()
+  shouldHaveSeperator: boolean = true;
+
+  @Input()
+  isDebrief: boolean = false;
+
+  public shouldShowFault(outcome: QuestionOutcome): boolean {
+    return outcome === 'DF';
```
Comment (matthew2564) [inline src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.ts]:
```
return outcome === CompetencyOutcome.DF;
```

Comment (RLCorp) [reply src/app/pages/view-test-result/components/safety-question-data-row/safety-question-data-row.ts]: addressed in newest commit

**Review by matthew2564 — APPROVED** [review]

**Pull request merged**